openapi.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Is this a generated file or are we making changes manually? This looks like the same information as in the `@Api` annotations so it would be preferable to maintain it in only one place.

Reply:
It would definitely be awesome if we could autogenerate this! I will probably leave that for future work though

Reply:
Let's at least get rid of the Google copyright and license. We can replace with our own if we want.